examples/persistence-service-catalog.md

```diff
@@ -16,19 +16,18 @@ sudo curl -L https://github.com/snowdrop/spring-boot-cloud-devex/releases/downlo
 sudo chmod +x /usr/local/bin/sb
 ```
 
-- odo's tool to create the service from the catalog
+- odo which will provide the ability to run the spring boot application on Openshift and also create the PostgreSQL database
 
 ```bash
 cd $GOPATH/src/github.com/redhat-developer
 git clone https://github.com/redhat-developer/odo.git && cd odo
-git fetch origin pull/622/head:pr-723
-git checkout pr-723
 make install && sudo cp $GOPATH/bin/odo /usr/local/bin
```

Comment:
Can we include the curl command to be used to install odo from a release (as next release should be out soon and include the code needed to play with this guide) AND mention if you want to build ourself the code, then follow these instructions ?

Reply:
Sure thing